problem_1/spec/multiples_of_3_5_spec.rb

```diff
@@ -1,79 +1,52 @@
 require'multiples_of_3_5'
 
-describe SumGenerator do
+describe MultiplesCalculator do
 
-	describe ".multiples_of_3" do
-		let (:counter) { SumGenerator.new(1...10) }
-		it "stores results of multiples_of_3" do 
+	describe ".multiples_generator" do
+		let (:calculator) { MultiplesCalculator.new(1...5) }
+		it "returns sum of multiples of 3 and 5 under 5" do 
 
-			result = counter.multiples_of_3
+			result = calculator.multiples_generator
 
-			expect(result.class).to be Array
-		end
-
-		it "finds all multiples of 3 under 10" do 
-
-			result = counter.multiples_of_3
-
-			expect(result).to eq [3, 6, 9]
-		end
-	end
-
-	describe ".multiples_of_3" do
-		let (:counter) { SumGenerator.new(1...100) }
-		it "finds all multiples of 3 under 100" do 
-
-			result = counter.multiples_of_3
-
-			expect(result).to eq [3,6,9,12,15,18,21,24,27,30,33,36,39,42,45,48,51,54,57,60,63,66,69,72,75,78,81,84,87,90,93,96,99]
-		end
-	end
-
-	describe ".multiples_of_5" do
-		let (:counter) { SumGenerator.new(1...10) }
-		it "finds all multiples of 5 under 10" do 
-
-			result = counter.multiples_of_5
-
-			expect(result).to eq [5]
+			expect(result).to eq 3
 		end
 	end
 
-	describe ".multiples_of_5" do
-		let (:counter) { SumGenerator.new(1...100) }
-		it "finds all multiples of 5 under 100" do 
+	describe ".multiples_generator" do
+		let (:calculator) { MultiplesCalculator.new(1...10) }
```

Comment:
Use `before(:each)` instead of declaring `let(:calculator)` for every test

Reply:
Looking back I think it makes sense not to use `before(:each)` because each you're initializing the class with different values ex: `1...10`. Generally `let` is used for multiple tests. You can just use a variable here instead of `let` 

`calculator = MultiplesCalculator.new(1...10)`